# 9D加速度センサGET
---
float64 accX
float64 accY
float64 accZ

